sdk/keyvault/keyvault-certificates/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 4.1.0-preview.2 (Unreleased)
 
+- Using `Poller` and `PollerLike` from the latest `core-lro` instead of relying on the temporary `KVPoller` and `KVPollerLike`.
```

Comment:
do you need to tweak the minimum dependency version of `core-lro` for this?

Reply:
I have updated all packages to use core-lro ^1.0.2 (otherwise rush won't let me be)